stable/locust/Chart.yaml

```diff
@@ -1,5 +1,11 @@
+apiVersion: v1
 name: locust
-description: A chart for locust distributed load testing
-version: 0.1.0
+description: A modern load testing framework
+version: 0.1.1
+maintainers:
+  - name: Vincent De Smet
```

Comment:
oh wow, didn't realise not including maintainers array caused monocular to break. sorry for that - I've finally added the discussed here.